test/docs/DependentLibs.md

```diff
@@ -0,0 +1,36 @@
+<!--
+Copyright (c) 2016, 2018 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# list of dependent libs
+
+  * asm-all-6.0_BETA.jar
+  * asmtools-6.0.jar
+  * commons-cli-1.2.jar
+  * commons-exec-1.1.jar
+  * javassist-3.20.0-GA.jar
+  * jcommander-1.48.jar
+  * junit-4.10.jar
+  * testng-6.10.jar
+
+To get the above dependent jars, please run `make getdependency`.
```

Comment:
It would not be common that you need to call this target on its own, as the getdependency target gets called when running...